pkg/cli/deployer/version.go

```diff
@@ -0,0 +1,25 @@
+package deployer
```

Comment:
why do we need to set version separately for deployer when it is just a symlink to oc?